content/posts/mongo_db_databases.md

```diff
@@ -0,0 +1,58 @@
+meta_og_url: "https://genezio.com/blog/mongo_db_databases/"
+meta_og_image: "https://genezio.com/images/ferretdb-genezio.webp"
+# meta data end
+customHeader: "White header"
+customFooter: "White footer"
+readTime: 10
+url: /blog/mongo_db_databases/
+---
+
+Hello everyone! We’re excited to share some big news: Genezio users can now set up MongoDB databases! Thanks to our new partnership with MongoDB Atlas, you can easily manage your MongoDB databases alongside your serverless functions. This means you get a powerful, fully managed database service that’s easy to use, scalable, and secure.
+
+## What Does This Mean for You?
+
+If you’re a Genezio user, you already know how simple it is to deploy serverless functions with just a few clicks. Now, with MongoDB hosting built into our platform, you can also quickly set up a NoSQL database to support your microservices, APIs, and apps. Here’s how this feature can improve your workflow:
+
+1. **Easy Integration**: With Genezio and MongoDB Atlas working together, you won’t have to deal with the hassle of setting up and managing a database. Whether your project is small or large, you can deploy MongoDB databases right from Genezio’s interface with just a few clicks.
+
+2. **Secure and Reliable**: Security is a top priority, especially when handling important data. MongoDB Atlas protects your information with top-notch security features like encryption, fine-grained access controls, and built-in backups. This keeps your data safe and always available.
+
+## How to Get Started
+
+To get started, simply log into your Genezio account and visit the [Databases section](https://app.genez.io/databases?utm_source=genezio&utm_medium=blog&utm_campaign=mongodb). From there you can create a new MongoDB database, get the connection string, and start using it in your projects. It’s that easy!
+
+The MongoDB database feature is also available in our YAML configuration file. You can now specify the database settings in your project’s YAML file, making it easy to manage your database alongside your serverless functions.
+
+```yaml
+databases:
+    - name: my-mongodb
+      type: mongo-atlas
+      region: us-east-1
+```
+
+Read more about the database feature in our [documentation](https://genezio.com/docs/features/databases/).
+
+## Express.js with MongoDB database Template
+
+We’ve put together a template that integrates Express.js with MongoDB, allowing you to easily set up a REST API backed by a MongoDB database. This template is ideal for building APIs, microservices, and more. Best of all, you can deploy it instantly with a single click by using the [deploy link](https://app.genez.io/start/deploy?repository=https://github.com/Genez-io/express-mongo-starter&utm_source=genezio&utm_medium=blog&utm_campaign=mongodb).
+
+You can find the source code for the template in our [GitHub repository](https://github.com/Genez-io/express-mongo-starter).
+
+## Conclusion
```

Comment:
## Conclusion: Start Using Genezio's MongoDB Integration Today

This new integration between Genezio and MongoDB Atlas is a game changer for developers like us. Now, you can manage your serverless functions and your NoSQL databases all from one unified platform. Whether you’re building small projects or scaling enterprise-level systems, you’ll have the tools you need to deploy secure, scalable, and reliable databases with ease.

Start using Genezio’s MongoDB integration today, and see how it can simplify your development process. With seamless database management, auto-scaling, and security features built-in, you can focus on delivering value to your users.

Keep an eye out for more updates, and happy coding!